### PR TITLE
CASMPET-5825: Add nexus space check to prereq

### PR DESCRIPTION
**Pull request by ndavidson-hpe**
# Description

Add in missing line to prereq

# Checklist Before Merging

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Comment by dzou-hpe** [comment]
@rustydb this is an example where we add more manual steps :(

File: upgrade/Stage_0_Prerequisites.md [hunk 1]
```diff
@@ -205,6 +205,10 @@ backup of Workload Manager configuration data and files is created. Once complet
    git push
    ```
 
+1. Check available space in Nexus, and free up space if needed.
```
Comment (dzou-hpe) [inline upgrade/Stage_0_Prerequisites.md:208]: i thought we had this as part of prereq script already? if not, we need to add it to prereq, no more manual steps into stage 0

**Review by dzou-hpe — CHANGES_REQUESTED** [review]

File: upgrade/Stage_0_Prerequisites.md (hunk 1 above)
Comment (ndavidson-hpe) [reply upgrade/Stage_0_Prerequisites.md:208]: It is a goss test. It was recommended to add to the prereq in https://github.com/Cray-HPE/docs-csm/pull/2073.

Comment (dzou-hpe) [reply upgrade/Stage_0_Prerequisites.md:208]: so we should just remove this? as we are checking it automatically already. the clean up should go to troubleshoot part. my point is that we should stop adding any more manual steps into the happy path

Comment (ndavidson-hpe) [reply upgrade/Stage_0_Prerequisites.md:208]: It will be removed. No need for this in the 1.3 steps

**Pull request closed without merging**